### PR TITLE
Update the Adding a New Page and the Switching Home pages

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request:
 - adds missing columns to the "Adding a New Page" page
 - adds missing columns to the "Switching Home Pages" page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/cms/page-add.html
https://docs.magento.com/user-guide/cms/page-home-new.html

### CODE REVIEW

File: src/cms/page-home-new.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ You can maintain a selection of different home pages, and activate the page that
 
 1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Default Pages** section and set **CMS Home Page** to the new page.
 
+   Other created pages can be used as the home page, such as:
```
Comment (dobooth) [inline src/cms/page-home-new.md:15]: I don't think we need this addition. It can be removed from the PR.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/cms/page-home-new.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/cms/page-home-new.md]: @dobooth 
Thanks, done ✅

File: src/cms/page-home-new.md [hunk 2]
```diff
@@ -12,6 +12,14 @@ You can maintain a selection of different home pages, and activate the page that
 
 1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Default Pages** section and set **CMS Home Page** to the new page.
 
+   - Welcome to our Exclusive Online Store
+   - Rewars Points
```
Comment (leslietilling) [inline src/cms/page-home-new.md:16]:
```suggestion
   - Rewards Points
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
minor typo

File: src/cms/page-home-new.md (hunk 2 above)
Comment (ZhovnirMaria) [reply src/cms/page-home-new.md]: Oh, thanks! Done.

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Review by leslietilling — APPROVED** [review]

**Pull request merged**